packages/fluxible-router/lib/createNavLinkComponent.js

```diff
@@ -1,13 +1,14 @@
 /**
- * Copyright 2015, Yahoo! Inc.
+ * Copyright 2015-Present, Yahoo! Inc.
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
-/*global window */
+/*global window,process */
 'use strict';
 var React = require('react');
 var RouteStore = require('./RouteStore');
 var debug = require('debug')('NavLink');
 var navigateAction = require('./navigateAction');
+var __DEV__ = process.env.NODE_ENV !== 'production';
```

Comment:
We have typically inlined this and used webpack to replace `process.env.NODE_ENV`.